packages/react-google-maps-api/src/useLoadScript.tsx

```diff
@@ -8,14 +8,26 @@ import { preventGoogleFonts } from './utils/prevent-google-fonts'
 import { makeLoadScriptUrl, LoadScriptUrlOptions } from './utils/make-load-script-url'
 
 import { defaultLoadScriptProps } from './LoadScript'
+import useIsMounted from './utils/useIsMounted'
+import { createUseAtMostOnce } from './utils/useAtMostOnce'
+import usePrevious from './utils/usePrevious'
 
 export interface UseLoadScriptOptions extends LoadScriptUrlOptions {
-  id?: string;
-  preventGoogleFontsLoading?: boolean;
+  id?: string
+  preventGoogleFontsLoading?: boolean
 }
 
 let previouslyLoadedUrl: string
 
+const useLoadScriptAtMostOnce = createUseAtMostOnce("useLoadScript should only be used at most once. The google script is loaded globally, and one hook's load could override another hook's load")
+
+const useIdNeverChange = (id: string) => {
+  const previousId = usePrevious(id) || id;
```

Comment:
I think we should actually remove that ID from options completely. What's the point of allowing to customize it?

Reply:
I have no id :)

I think we are not yet clear on what's the best solution to load this lib correctly. Shouldn't we pursue discussions in issues?